app/models/comment.rb

```diff
@@ -1,3 +1,7 @@
 class Comment < ActiveRecord::Base
   belongs_to :post
+  belongs_to :user
+
+  validates :body, length: { minimum: 5 }, presence: true
+  validates :user_id, presence: true
```

Comment:
Following Rails conventions, you can write this line as:

```ruby
validates :user, presence: true
```

They are both technically the same, but you normally see the one above for association validations. :smiley: